opensrp-app/res/values/dimens.xml

```diff
@@ -1,132 +1,134 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
```

Comment:
Not sure why we have this large diff here?

Reply:
same as above

Reply:
https://github.com/OpenSRP/opensrp-client-core/pull/715#discussion_r551863135